sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -28,13 +28,28 @@ static StorageTestBase()
 #endif
         }
 
+        /// <summary>
+        /// Add a static TestEventListener which will redirect SDK logging
+        /// to Console.Out for easy debugging.
+        ///
+        /// This is only here to run before any of our tests make requests.
+        /// </summary>
+        private readonly TestEventListener _listener;
+
         public StorageTestBase(bool async, RecordedTestMode? mode = null)
             : base(async, mode ?? RecordedTestUtilities.GetModeFromEnvironment())
         {
+            _listener = new TestEventListener();
```

Comment:
Do we want one of these per test class?  Or one per run?  Consider making it `static` and putting it in the `.cctor`.

Reply:
Also - does this slow down the rest run?  Would we want to only enable this when a debugger is attached?  Or is this worth it to diagnose failures on the CI?

Reply:
We already have a check for this in TestEventListener.LogEvent